### ISSUE TITLE
invalid macro in manpage

### ISSUE THREAD

**Issue by kilobyte** [MestreLion/git-tools#20]
```
manpage-has-errors-from-man usr/share/man/man1/git-clone-subset.1.gz 54: warning: macro `!(<pattern>)'' not defined
```

**Comment by gebailey** [comment]
This appears to be an error from something called lintian, which seems to be pretty Debian specific.  Is there a way to perform validation on a non-Debian system?  The manpage appears to render OK on CentOS 7.  man --warnings git-clone-subset doesn't emit any errors that I can see.

**Comment by MestreLion** [comment]
Nice catch @kilobyte , thanks!

@gebailey : The problem was the character `'` (from `'!(<pattern>)'`') appearing in the beginning of a line, it is interpreted by man as a macro. Lintian is just tool to find common errors such as this one. The tool may be Debian-specific, but here the page did not render correctly: the line `'!(<pattern>)' rule. Quote it or escape it on command line, so it` was missing from the rendered page.

The solution, was quite trivial, as suggested by https://lintian.debian.org/tags/manpage-has-errors-from-man.html : *reformat the paragraph so that these characters are not at the start of a line*

**Comment by kilobyte** [comment]
Alas, the new version now has a line starting with 'git ...
The fix is obvious.

**Comment by MestreLion** [comment]
Thanks again @kilobyte. Found and fixed that one and other similar occurrences in commit ea095192bc4180cc89d56f650deaeb0cbfe1ae25

By the way.. I noticed you dropped all your debian manpages in favor of these. Are they really equivalent? I'm completely unfamiliar with manpage (nroff?) syntax, so I'm unable to compare them, but if you want to combine the best features of each version, feel free to open a pull request.
